packages/common/src/content/contentUtils.ts

```diff
@@ -453,44 +466,37 @@ const getContentRelativeUrl = (
 export async function getServiceStatus(
   entity: IHubEditableContent,
   options: IGetServiceStatusOptions
-): Promise<IHubServiceBackedContentStatus> {
+): Promise<IHubContentStatus> {
   // get the request options for the `getService` call, and set a default timeout if one is not provided
   const { timeout = 3000, ...requestOptions } = options;
-  const unavailable = {
-    kind: "service",
-    service: {
-      availability: "unavailable",
-    },
-  } as IHubServiceBackedContentStatus;
+  const isServiceBackedEntity = isService(entity.url.split("?")[0]); // remove any query params
 
-  if (entity.url) {
+  if (isServiceBackedEntity) {
     // set up our two promises: one to get the service definition and one to sleep for 3 seconds
-    const definitionPromise: Promise<IFeatureServiceDefinition> = getService({
+    const definitionPromise = getService({
       url: entity.url,
       authentication: requestOptions.authentication,
-    });
-    const timeoutPromise = (ms: number) => {
-      return new Promise((resolve) => setTimeout(resolve, ms));
-    };
-
-    // race the two promises
-    const status = Promise.race([definitionPromise, timeoutPromise(timeout)])
-      .then((result) => {
-        return {
-          kind: "service",
-          service: {
-            // if we got a result, the service is available, otherwise it's slow
-            availability: result ? "available" : "slow",
-          },
-        };
+    })
```

Comment:
Any reason we wouldn't use the rest of `requestOptions` and not just `authentication`?

```ts
const { url } = entity;
const definitionPromise = getService(...requestOptions, url);
```